package.json

```diff
@@ -4,17 +4,24 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
+    "bcrypt": "^5.0.1",
+    "body-parser": "^1.20.0",
+    "dotenv": "^16.0.1",
     "express": "^4.18.1",
+    "express-session": "^1.17.2",
+    "jsonwebtoken": "^8.5.1",
     "mongoose": "^6.3.3",
     "ts-node": "^10.7.0",
     "typescript": "^4.6.4"
   },
   "scripts": {
     "start": "node dist/index.js",
     "build": "tsc",
-    "dev": "ts-node src/index.ts"
+    "dev": "nodemon src/index.ts"
```

Comment:
we aren't using javascript we are using typescript, ts-node setup is already done use that